backend/rest/rest.go

```diff
@@ -76,7 +81,10 @@ func (c *client) Search(query, lang string) []*backends.Result {
 		fmt.Println(err)
 	}
 
-	cachedRepoIds := initializeCache(values)
+	cachedRepoIds := make([]int64, 0, 0)
```

Comment:
I can't comment on line 79. Stupid git. Anyway:
if we just cleared the repo, we also have no reason to try and search the db. (storage.get())